app/helpers/template_helper.rb

```diff
@@ -0,0 +1,14 @@
+  module TemplateHelper
+    def build_remediation_plan(remediation_ids, host)
+      entries = PreupgradeReportEntry.where(id: remediation_ids.split(','), host_id: host.id)
+      result = ""
+
+      entries&.each do |entry|
+        JSON.parse(entry.detail.gsub('=>',':'))['remediations']&.each do |remediation|
```

Comment:
Do you have an example what `entry.detail` looks like?

Reply:
```
{"related_resources"=>[{"scheme"=>"package", "title"=>"leapp"}, {"scheme"=>"package", "title"=>"leapp-deps"}, {"scheme"=>"package", "title"=>"leapp-repository"}, {"scheme"=>"package", "title"=>"leapp-repository-deps
"}, {"scheme"=>"package", "title"=>"python2-leapp"}, {"scheme"=>"package", "title"=>"snactor"}], "remediations"=>[{"type"=>"command", "context"=>["yum", "-y", "remove", "leapp leapp-deps leapp-repository leapp-repository-deps python2-leapp snactor"]}]}
```

Reply:
So it is a pretty-printed ruby hash with rocket syntax as a string?

Reply:
Yeah, I missed that somehow. I will update the way how the data are saved in place first.